src/components/Flag/Flag.tsx

```diff
@@ -27,11 +28,24 @@ const getSourceByIsoCode = (isoCode: MuiTelInputCountry | null) => {
   return ''
 }
 
-const Flag = ({ size = 'small', isoCode, countryName = '' }: FlagProps) => {
+const defaultGetFlagSources: GetFlagSources = (isoCode, width) => {
+  return {
+    png: `https://flagcdn.com/w${width}/${isoCode}.png`,
+    webp: `https://flagcdn.com/w${width}/${isoCode}.webp`
+  }
+}
+
+const Flag = ({
+  size = 'small',
+  isoCode,
+  countryName = '',
+  getFlagSources = defaultGetFlagSources
```

Comment:
I'm not sure, but I would say you break the `memo` wrapper as the `getFlagSources` get a new reference on every render

Reply:
I'm also not 100% sure, but i don't think this is the case. As how i see this: `defaultGetFlagSources` is defined outside the component, so it will only be defined once on build time and will never get a new reference.

Reply:
Yes but `getFlagSources` if it's defined, will get a new reference on every render in user app

Reply:
Hmm, ok, this is out of my knowledge.. Can you suggest a solution for this?

Reply:
I think we can let this as it is, and provide more information in the documentation

Reply:
Ok, then i will leave it as is. Provide more informatie? Updating the API Reference is that enough? (that what I just did). Let me know if you miss something.